modules/nlu/src/backend/election/natural-election.test.ts

```diff
@@ -0,0 +1,67 @@
+import '../../../../../packages/bp/src/sdk/botpress'
+import { extractElectedIntentSlot } from './natural-election'
+import { fakeEventUnderstanding } from '../../test/types'
+
+describe('naturalElection', () => {
+  describe('extractElectedIntentSlot', () => {
+    it("should return input directly if the 'predictions' property is nil", () => {
+      const input = fakeEventUnderstanding({ predictions: undefined })
```

Comment:
To me, it seems weird that you can actually build a predictions obj with an undefined value, compiler should warn you if tyings are ok

Reply:
Not sure what to do with this. Are you asking me to change something?

Reply:
It's just a question